docs/contributing.rst

```diff
@@ -29,27 +31,44 @@ Fix bugs
 --------
 
 Check out the `open bugs <https://github.com/Kinto/kinto/issues>`_ - anything
-tagged with the **[easy-pick]** label could be a good choice for newcomers.
+tagged with the |easy-pick label|_ could be a good choice for newcomers.
+
+.. |easy-pick label| replace:: **easy-pick** label
```

Comment:
Like I wrote in https://github.com/Kinto/kinto/pull/574/commits/e46ba29e2875a6e3c59bdf06d453f12e46d338ff, it's kind of annoying, and I'm not sure it looks good in rendered HTML, but sure :)